build-image.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 set -e
-sudo docker build --tag strip-coverletter .
```

Comment:
`sudo` isn't necessary on AWS instances, while locally you can avoid it as weel by adding your user to the `docker` group.